OpenRA.Mods.Common/ServerTraits/MasterServerPinger.cs

```diff
@@ -118,14 +124,31 @@ void UpdateMasterServer(S server, int numPlayers, int numSlots, int numBots, int
 						if (isInitialPing)
 						{
 							var masterResponseText = Encoding.UTF8.GetString(masterResponse);
+							Log.Write("server", "Master server: " + masterResponseText);
```

Comment:
If we consider now empty server response possible, is it useful to log it? It could be moved below into if.